src/components/Form/FormSection.jsx

```diff
@@ -38,6 +47,7 @@ const FormSection = ({ title, state, statusType, defaultMessage, file = null, ch
       {title}
     </Typography>
     {children}
+    {showSpinner && <CircularProgress />}
```

Comment:
Quick question, what could this spinner be used for?

And if there's a reason for this, how would this be different from utilizing the `LoadingAnimation` component with the circular spinner?